src/lib/I18n.js

```diff
@@ -83,19 +97,23 @@ class I18n {
     /**
      * @param {string} language - ISO 639-1 language codes, e.g. en, en-us, de, de-at
      */
-    setLanguage(language) {
+    static set language(language) {
         const languageToUse = this.getClosestSupportedLanguage(language);
         if (languageToUse !== language) {
             console.warn(`Language ${language} not supported, using ${languageToUse} instead.`);
         }
         this._language = languageToUse;
     }
 
-    get language() {
+    static get language() {
         return this._language;
     }
 
-    get dictionary() {
+    static get dictionary() {
         return this._dict;
     }
-}
+
+    static get fallbackLanguage() {
+        return this._fallbackLanguage || 'en';
```

Comment:
set _fallbackLanguage to 'en' in constructor if fallbackLanguage was not provided instead of the " || 'en'" here